### DISCUSSION TITLE
Add something like Views in GA

### DISCUSSION THREAD

**Discussion by slubbodan** [plausible/analytics#1058] [Anything Else]
I've searched and found some similar requests on filtering out a specific part of a website, like /blogs/.
My case is a little different though and i havent found a way to deal with it.
Basically my site contains both our external web and our intranet. the intranet being located in /intranet/.
So, what i've done in GA is to have 2 separate Views:
External web (excluding any traffic to /intranet/**)
Intranet (excluding any traffic that isn't to the /intranet/-part of the site)
Using Pageview Goals doesn't help, since i need to view the statistics for each part like it was it's own website. Right now all the numbers for the external site includes our intranet visits as well.

**Comment by metmarkosaric** [comment]
Did you consider to add them as individual sites in Plausible? That would give you two different dashboards with independent stats separated from each other.

**Reply by slubbodan** [reply]
Yes but I failed. :-) How could i not include the /intranet/ part in such a way? And to add a site with this url (nai.uu.se/intranet/) didn't work. I mean i could only have one <script>-snippet right? 
the site is nai.uu.se

**Reply by metmarkosaric** [reply]
You can add subfolders as sites in Plausible. So you could add nai.uu.se/intranet/ and have a unique snippet on that subfolder only. 

We also have other functionality that may help do what you need:

Send stats to multiple dashboards https://plausible.io/docs/plausible-script#can-i-send-stats-to-multiple-dashboards-at-the-same-time

Exclude specific pages/sections from being tracked https://plausible.io/docs/excluding-pages

**Reply by slubbodan** [reply]
Ok thanks I'll try this!

**Reply by slubbodan** [reply]
PS If i exclude using the different script snippet, can i at the same time track outbound links (that is also another version of the script)

**Reply by metmarkosaric** [reply]
Yes, there are instructions on the link I sent https://plausible.io/docs/excluding-pages#3-change-the-snippet-on-your-site-to-the-new-snippet
